nodejs/v3/express-mongoose/app/src/appsignal.ts

```diff
@@ -0,0 +1,10 @@
+import { Appsignal } from "@appsignal/nodejs"
+
+export default new Appsignal({
```

Comment:
```suggestion
new Appsignal({
```
Export not needed, right?